initiatives/2019-09-credsperiment-alpha.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "type": "sourcecred/initiativeFile",
+    "version": "0.2.0"
+  },
+  {
+    "title": "CredSperiment Alpha",
+    "timestampIso": "2019-09-15T00:00:00.000Z",
+    "weight": { "incomplete": 0, "complete": 0 },
+    "completed": false,
+    "champions": ["https://discourse.sourcecred.io/u/decentralion"],
+    "dependencies": {
+      "urls": [
+        "https://discourse.sourcecred.io/t/create-identity-plugin/662",
+        "https://discourse.sourcecred.io/t/create-discourse-plugin/660"
+      ],
+      "entries": [
+        { "title": "Placeholder: Create GitHub plugin" },
+        { "title": "Placeholder: Create Scores API" },
+        { "title": "Placeholder: Develop the Grain concept" }
```

Comment:
GitHub and Grain weren't mentioned as dependencies yet in the meeting. But seem intuitive to me as dependencies.

Reply:
Good points. Also, "TimelineCred" was a dependency, and original PageRankCred was a dep of timeline cred.